src/Form/PartnerType.php

```diff
@@ -115,7 +150,9 @@ function ($tagsAsArray) {
             },
             function ($tagsAsString) {
                 // transform the string back to an array
-                return null !== $tagsAsString ? explode(',', $tagsAsString) : [];
+                $pattern = '/(\s*,*\s*)*,+(\s*,*\s*)*/';
```

Comment:
Je ne comprends pas pourquoi Sonar me fait une alerte "[Rework this part of the regex to not match the empty string.](https://sonarcloud.io/project/issues)" sur la regex, car justement je veux virer les éventuelles blancs entre virgules ou à la fin pour ne pas créer un code vide

Reply:
j'ai ignoré ces erreurs en faux-positif

Reply:
Oui, il existe une option pour l'ignorer